command/AudioFileHelper.js

```diff
@@ -55,36 +54,62 @@ class CommandHelper {
     return false;
   }
 
-  async storeAudioFileLocation(articleId, summaryOnly, location) {
+  //DONE
```

Comment:
various //DONE markers in this file to clean out

Reply:
fixed.